pinlockview/src/main/java/com/andrognito/pinlockview/PinLockAdapter.java

```diff
@@ -189,8 +204,49 @@ private void configureDeleteButtonHolder(DeleteViewHolder holder) {
                         mCustomizationOptionsBundle.getButtonSize());
                 holder.mDeleteButton.setLayoutParams(params);
             } else {
+                holder.mDeleteButton.setEnabled(false);
                 holder.mDeleteButton.setVisibility(View.GONE);
             }
+            LinearLayout.LayoutParams params = new LinearLayout.LayoutParams(
+                    mCustomizationOptionsBundle.getButtonSize(),
+                    mCustomizationOptionsBundle.getButtonSize());
+            holder.mDeleteButton.setLayoutParams(params);
+        }
+    }
+
+    private void configureEnterButtonHolder(EnterViewHolder holder) {
+        if (holder != null) {
+            if (mCustomizationOptionsBundle.isShowEnterButton()) {
+                // Set Enable of Enter Button
+                if (mPinLength >= mCustomizationOptionsBundle.getPinLength()) {
+                    holder.mEnterButton.setEnabled(true);
+                    if (mCustomizationOptionsBundle.isUseCustomEnterButtonImages()) {
```

Comment:
instead of nesting so many `if`'s could we convert some of these to guard statements? Looks like at least the null check could be built that way